src/main.ts

```diff
@@ -19,7 +19,12 @@ const networkConfigUrl = (() => {
   }
 })();
 
-const walletParam = (agoricNetName == "main" ?  '' : '?wallet=main');
+const redirectParams = new URLSearchParams();
```

Comment:
Let's make this into a function that returns either `'?' + redirectParams.toString()` or empty string `''`. Then below call use the return value of the function so that we don't always put `?` on the URL if there's no params.

Reply:
is a ternary ok?